operations/template/alert.tf

```diff
@@ -28,6 +28,47 @@ resource "azurerm_monitor_action_group" "notify_slack_email" {
   }
 }
 
+resource "azurerm_monitor_activity_log_alert" "azure_service_health_alert" {
+  count               = local.non_pr_environment ? 1 : 0
+  name                = "cdcti-${var.environment}-azure-status-alert"
+  location            = "global"
```

Comment:
This should be `data.azurerm_resource_group.group.location`?  For every other resource that asks for a `resource_group_name` and `location`, we've put `data.azurerm_resource_group.group.location` for the `location`.